orchestra/utils/view_helpers.py

```diff
@@ -12,3 +13,84 @@ class IsAssociatedWorker(permissions.BasePermission):
     def has_object_permission(self, request, view, obj):
         worker = Worker.objects.get(user=request.user)
         return obj.worker == worker
+
+
+def _get_changed_fields(old_todo, new_todo):
```

Comment:
where was all this logic in the original code?

Reply:
Left only `title` and `description` part.